.travis.yml

```diff
@@ -1,29 +1,24 @@
 matrix:
   include:
-  - name: "DApp unit tests"
+  - name: "origin-contracts / origin-js / origin-dapp unit tests"
     language: node_js
-    node_js: 8
+    node_js: 10
```

Comment:
+1 for upgrading to node 10.
Now, if we use it for testing, we should also switch our prod containers to using it. See for example [here](https://github.com/OriginProtocol/origin/blob/master/deployment/dockerfiles/event-listener#L3) and in other containers in that directory.